README.md

```diff
@@ -38,6 +38,15 @@ Upon first use, the plugin will download the advice dataset from `https://scala-
 `$HOME/.clippy` directory. The dataset will be updated at most once a day, in the background. You can customize the
 dataset URL and local store by using the `-P:clippy:url=` and `-P:clippy:store=` compiler options.
 
+# Enabling colors
+
+Clippy enhances some compiler errors with ANSI colors. If you'd like to enable colors on your terminal, you can
+`-P:clippy:colors=true` compiler option. To enable globally, add following line to `~/.sbt/0.13/clippy.sbt`:
```

Comment:
"you can ... *add*" (missing I think). Also the "enable globally" applies to the case when using the SBT plugin, while the `-P` when using the compiler plugin directly